.github/workflows/build-test.yml

```diff
@@ -287,7 +287,7 @@ jobs:
 
       - name: Remove Duplicate Declaration Warnings
         run: |
-          sed -i "6559d" /opt/hostedtoolcache/Python/3.8.6/x64/lib/python3.8/site-packages/sphinx/domains/cpp.py
+          sed -i "6559d" /opt/hostedtoolcache/Python/3.8.6/x64/lib/python3.8/site-packages/sphinx/domains/cpp.py || sed -i "6559d" /opt/hostedtoolcache/Python/3.8.7/x64/lib/python3.8/site-packages/sphinx/domains/cpp.py
```

Comment:
once 3.8.8 gets released, this will most likely break

Reply:
We could make this more robust by utilizing something like:

```
SITE_PACKAGES_DIR=$(python3 "-c" "from distutils import sysconfig; print(sysconfig.get_python_lib())")
```